MidCamp/midcamp
# Upgrade Status

# Description

Adds and enables the Upgrade Status module.  Also updates Drupal Console and some of its dependencies, which were sufficiently outdated that they were preventing the installation of Upgrade Status.

# Test instructions

- Install dependencies with `lando composer install`
- Import configuration with `lando drush cim -y`
- Visit [`/admin/reports/upgrade-status`](http://midcamp-org.lndo.site/admin/reports/upgrade-status) and observe the Upgrade Status module is enabled
